docs/data/material/components/autocomplete/InfiniteScrolling.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import TextField from '@mui/material/TextField';
+import Autocomplete from '@mui/material/Autocomplete';
+
+export default function InfiniteScrolling() {
```

Comment:
In this demo in Material UI, when you scroll to the bottom and the loading text is visible it expands the listbox, whereas in Joy UI it doesn't happen like that.

Reply:
I believe it is because the JSX was moved. See https://github.com/mui/material-ui/pull/35653#discussion_r1133542955.